package.json

```diff
@@ -10,7 +10,7 @@
     "build:accept": "yarn esbuild:generic accept/src/index.ts --outfile=accept/dist/index.js",
     "build:commit-status": "yarn esbuild:generic commit-status/src/index.ts --outfile=commit-status/dist/index.js",
     "build:db": "yarn esbuild:generic db/src/index.ts --outfile=db/dist/index.js",
-    "build:deploy-image": "yarn esbuild:generic deploy-image/src/index.ts --outfile=deploy-image/dist/index.js",
+    "build:push-image-ghcr": "yarn esbuild:generic push-image-ghcr/src/index.ts --outfile=push-image-ghcr/dist/index.js",
```

Comment:
nit: order alphabetically

Reply:
I called it `push-image-ghcr` and specifically set it up to push images with a tag of <BRANCH_NAME> to GHCR. 

My proposal is to make a second action with a name like `push-image-dockerhub` that handles pushing realeases and their corresponding tags to dockerhub.

A build-image action sounds like a good addition, I'm not sure about persisting it without pushing though. I do like the idea of moving to smaller actions with less inputs. And defining more complex sequences of tasks in the workflows.

Reply:
fixed the ordinality

Reply:
I'm not a fan of two actions doing the same thing only with different parameters. Registry feels more like a variable than a different function IMO. This also allows us to change e.g. the default registry without having to reach into each individual component workflow.

Reply:
> My proposal is to make a second action with a name like push-image-dockerhub that handles pushing realeases and their corresponding tags to dockerhub.

I'm not a fan of two actions doing the same thing only with different parameters. Registry feels more like a variable than a different function IMO. This also allows us to change e.g. the default registry without having to reach into each individual component workflow.

> A build-image action sounds like a good addition, I'm not sure about persisting it without pushing though. I do like the idea of moving to smaller actions with less inputs. And defining more complex sequences of tasks in the workflows.

A good level of abstraction for me is the level of detail consumers would normally be concerned with (e.g. the `db` action which does several steps but consumers are only interested in the compound effect, not the steps). Is there a use case where components want to separate building and pushing images? Alternatively we could have separate actions and one 'composite' action tying these together for easier consumption (e.g. https://github.com/exivity/actions/blob/main/process-binary/action.yml)